docs/api/data_types.md

```diff
@@ -16,16 +16,37 @@
             - num_completion_tokens
 
 ::: llmengine.CompletionSyncResponse
+    selection:
```

Comment:
hmm, do we must list these one by one?

Reply:
There's a configuration in `mkdocs.yml` that allows members to be listed automatically, but the trade off is that we have no control over the order of the members.